corehq/apps/app_manager/views/modules.py

```diff
@@ -604,6 +606,13 @@ def _new_shadow_module(request, domain, app, name, lang):
     return back_to_main(request, domain, app_id=app.id, module_id=module.id)
 
 
+def _new_training_module(request, domain, app, name, lang):
```

Comment:
Could we change the default name for the forms that you add to a Training module (from `Survey` to something else)? I think it would help emphasize the distinction and help users understand the purpose of a Training module. I'm not quite sure what the best alternative is, but maybe `Lesson`?

Reply:
Changing the small icon that shows in the side bar next to training modules (from the folder icon to something else) could be cool too, but less important I think. Again I'm not certain what would be best, but maybe a small notebook?

Reply:
Updated icon:
<img width="444" alt="screen shot 2018-04-05 at 11 41 06 am" src="https://user-images.githubusercontent.com/1757035/38376365-6605b5c8-38c6-11e8-8fae-18f29839d266.png">

Reply:
Updated default form name:
<img width="439" alt="screen shot 2018-04-05 at 11 41 13 am" src="https://user-images.githubusercontent.com/1757035/38376385-763c3840-38c6-11e8-9398-a81599fecba6.png">

Reply:
I also removed the child module menu from training modules, as discussed.